Tasks/PhoneGapCommand/task.json

```diff
@@ -0,0 +1,72 @@
+{
+    "id": "61EC6339-4A6A-4A69-875E-5A6f4DB157A2",
+    "name": "PhoneGapCommand",
+    "friendlyName": "PhoneGap Command",
+    "description": "Run a PhoneGap CLI command",
+    "helpMarkDown": "[More Information](http://go.microsoft.com/fwlink/?LinkID=692058)",
+    "category": "Build",
+    "visibility": [
+                "Build"
+                  ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 1
+    },
+    "demands": [
+        "npm"
+    ],
+    "instanceNameFormat": "PhoneGap Command $(phonegapCommand)",
+     "groups": [
+        {
+            "name":"advanced",
+            "displayName":"Advanced",
+            "isExpanded":false
+        }
+    ],
+    "inputs": [
+        { 
+            "name": "phonegapCommand", 
+            "type": "string", 
+            "label": "Command", 
+            "defaultValue":"", 
+            "required":true, 
+            "helpMarkDown": "PhoneGap command to run. Ex: \"platform add android\""
+        },
+        { 
+            "name": "phonegapArgs", 
+            "type": "string", 
+            "label": "Arguments", 
+            "required":false,
+            "helpMarkDown": "Arguments that should be used for the command."
+        },
+        { 
+            "name": "phonegapVersion", 
+            "type": "string", 
+            "label": "PhoneGap Version", 
+            "required":false, 
+            "helpMarkDown": "Version of PhoneGap tools to use to run the command. If not specified, looks at taco.json, the CORDOVA_DEFAULT_VERSION env var, and finally falls back to the latest version."
```

Comment:
CORDOVA_DEFAULT_VERSION should be PHONEGAP_DEFAULT_VERSION here and we should remove the reference to taco.json.